src/chrome/content/rules/pvv.ntnu.no.xml

```diff
@@ -0,0 +1,19 @@
+<ruleset name="Programvareverkstedet">
+	<target host="http://pvv.org" />
+	<target host="http://pvv.ntnu.no" />
+	<target host="http://www.pvv.org" />
+	<target host="http://www.pvv.ntnu.no" />
+	<target host="http://mail.pvv.org" />
+	<target host="http://mail.pvv.ntnu.no" />
+	<target host="http://webmail.pvv.org" />
+	<target host="http://webmail.pvv.ntnu.no" />
+	<target host="http://idp.pvv.org" />
+	<target host="http://idp.pvv.ntnu.no" />
+	<target host="http://list.pvv.org" />
+	<target host="http://list.pvv.ntnu.no" />
+
+	<rule from="^http://(.+\.)?pvv\.org" to="https://$1.pvv.ntnu.no" />
```

Comment:
Please try use use a stricter regex pattern, such as:

```suggestion
	<rule from="^http://(\w+\.)?pvv\.org" to="https://$1pvv.ntnu.no" />
```

You also have an extra `.` in your `to`.